proto/build.gradle

```diff
@@ -43,6 +44,43 @@ protobuf {
     }
 }
 
+def protoVersion = "0.6.0"
+def protoArchivePath = "$buildDir/archives/opentelemetry-proto-${protoVersion}.zip"
+
+def downloadProtoArchive = tasks.register("downloadProtoArchive", Download) {
+    onlyIf { !file(protoArchivePath).exists() }
+    src "https://github.com/open-telemetry/opentelemetry-proto/archive/v${protoVersion}.zip"
+    dest protoArchivePath
+}
+
+def verifyProtoArchive = tasks.register("verifyProtoArchive", Verify) {
+    dependsOn(downloadProtoArchive)
+    src protoArchivePath
+    algorithm "SHA-256"
+    // To generate checksum, download the file and run "shasum -a 256 ~/path/to/vfoo.zip"
```

Comment:
should we include something in the CONTRIBUTING.md about the process of updating the proto version?